testhelper/certificatetest.go

```diff
@@ -71,20 +71,20 @@ func setupCertificates() {
 	// This will be flagged by the static code analyzer as 'Use of a weak cryptographic key' but its intentional
 	// and is used only for testing.
 	k, _ := rsa.GenerateKey(rand.Reader, 1024)
-	unsupported = getRSACertTupleWithPK(k, "Notation Unsupported Root", nil)
+	unsupported = GetRSACertTupleWithPK(k, "Notation Unsupported Root", nil)
 }
 
-func getCertTuple(cn string, issuer *CertTuple) CertTuple {
+func getCertTuple(cn string, issuer *RSACertTuple) RSACertTuple {
 	pk, _ := rsa.GenerateKey(rand.Reader, 3072)
-	return getRSACertTupleWithPK(pk, cn, issuer)
+	return GetRSACertTupleWithPK(pk, cn, issuer)
 }
 
 func getECCertTuple(cn string, issuer *ECCertTuple) ECCertTuple {
 	k, _ := ecdsa.GenerateKey(elliptic.P384(), rand.Reader)
-	return getECDSACertTupleWithPK(k, cn, issuer)
+	return GetECDSACertTupleWithPK(k, cn, issuer)
 }
 
-func getRSACertTupleWithPK(privKey *rsa.PrivateKey, cn string, issuer *CertTuple) CertTuple {
+func GetRSACertTupleWithPK(privKey *rsa.PrivateKey, cn string, issuer *RSACertTuple) RSACertTuple {
```

Comment:
Why do we need to export these functions?

Reply:
These are used in notation-go-lib for unit tests.